app/views/rent_mailer/new_rent_email.text.erb

```diff
@@ -0,0 +1,3 @@
+You have registered a new rent for the book <%= @rent.book.title %>
```

Comment:
You do not need this `ERB` file, you ever have the silm one ;)

Reply:
The style guided you sent to me says we should create both versions html and plain text:
https://github.com/rubocop-hq/rails-style-guide#mailers

Also I have read in some blogs that always sending the plain text version of the email is a good practice. For example when you receive an email in your mobile device the notification shows the text plain version of the email.

Reply:
Your right!! I just not see the `.txt` before the `.erb`. Mala mia!